Sources/App/Utilities/SemverVersion.swift

```diff
@@ -1,14 +1,33 @@
 struct SemverVersion {
     enum Bump {
+        case stage
         case major
         case minor
         case patch
     }
 
     struct Prerelease {
+        enum Identifier: String {
+            case alpha
+            case beta
+            case rc
+        }
+        
         var name: String
         var major: Int?
         var minor: Int?
+        
+        var bump: Self? {
```

Comment:
Maybe `bumped` ? Or `nextRelease`? A verb for a property seems strange. `bump()` would be better, too.